.ci/pytorch/smoke_test/smoke_test.py

```diff
@@ -396,7 +396,7 @@ def smoke_test_nvshmem() -> None:
     except ImportError:
         # Not built with NVSHMEM support.
         # torch is not compiled with NVSHMEM prior to 2.9
-        if torch.__version__ < "2.9":
```

Comment:
We are trying to compare different types of string:
```
>>> "2.10.0.dev20250910+cu128" < "2.9"
True
>>> "2.10.0.dev20250910+cu128" < "2.9.*"
True
>>> 
``

Reply:
I believe python has a way to convert string "2.9" into a version object?